packages/react/src/components/InlineLoading/InlineLoading.tsx

```diff
@@ -93,26 +93,34 @@ const InlineLoading = ({
 
   const getLoading = () => {
     let iconLabel = iconDescription ? iconDescription : status;
+    let element = undefined;
+    let loading = {
+      text: description ? (
+        <div className={`${prefix}--inline-loading__text`}>{description}</div>
+      ) : undefined,
+      animation: undefined,
+    };
+
     if (status === 'error') {
-      return (
+      element = (
```

Comment:
Per https://github.com/carbon-design-system/carbon/issues/19467, if the issue only affects the inactive status, why modify the handling for all other statuses? Is the issue more pervasive than reported?